Can't build on Debian 9

**Describe the bug**
I am trying to build latest master (7ae9d4a900ee8cadcf44606f) in Debian 9 environment (for building AppImage packages),
Got the following error:

```
make[4]: Entering directory '/build/packet/linux-64/synfigstudio-master/build/synfig/synfig-studio/src/gui'
  CXX      states/synfigstudio-state_brush.o
In file included from ../../src/brushlib/brushlib.hpp:5:0,
                 from ../../src/brushlib.h:33,
                 from ../../src/synfigapp/actions/layerpaint.h:38,
                 from states/state_brush.cpp:56:
../../src/brushlib/brush.hpp: In member function 'bool brushlib::Brush::stroke_to(brushlib::Surface*, float, float, float, float, float, double)':
../../src/brushlib/brush.hpp:618:20: error: call of overloaded 'abs(float&)' is ambiguous
       if (abs(xtilt) > abs(ytilt)) {
                    ^
In file included from /usr/include/c++/6/cstdlib:75:0,
                 from /usr/include/c++/6/bits/stl_algo.h:59,
                 from /usr/include/c++/6/algorithm:62,
                 from ./_smach.h:38,
                 from ./smach.h:33,
                 from ../../src/gui/states/state_brush.h:33,
                 from states/state_brush.cpp:36:
/usr/include/stdlib.h:735:12: note: candidate: int abs(int)
 extern int abs (int __x) __THROW __attribute__ ((__const__)) __wur;
            ^~~
In file included from /usr/include/c++/6/bits/stl_algo.h:59:0,
                 from /usr/include/c++/6/algorithm:62,
                 from ./_smach.h:38,
                 from ./smach.h:33,
                 from ../../src/gui/states/state_brush.h:33,
                 from states/state_brush.cpp:36:
/usr/include/c++/6/cstdlib:185:3: note: candidate: __int128 std::abs(__int128)
   abs(__GLIBCXX_TYPE_INT_N_0 __x) { return __x >= 0 ? __x : -__x; }
   ^~~
/usr/include/c++/6/cstdlib:180:3: note: candidate: long long int std::abs(long long int)
   abs(long long __x) { return __builtin_llabs (__x); }
   ^~~
/usr/include/c++/6/cstdlib:172:3: note: candidate: long int std::abs(long int)
   abs(long __i) { return __builtin_labs(__i); }
   ^~~
In file included from ../../src/brushlib/brushlib.hpp:5:0,
                 from ../../src/brushlib.h:33,
                 from ../../src/synfigapp/actions/layerpaint.h:38,
                 from states/state_brush.cpp:56:
../../src/brushlib/brush.hpp:618:33: error: call of overloaded 'abs(float&)' is ambiguous
       if (abs(xtilt) > abs(ytilt)) {
                                 ^
In file included from /usr/include/c++/6/cstdlib:75:0,
                 from /usr/include/c++/6/bits/stl_algo.h:59,
                 from /usr/include/c++/6/algorithm:62,
                 from ./_smach.h:38,
                 from ./smach.h:33,
                 from ../../src/gui/states/state_brush.h:33,
                 from states/state_brush.cpp:36:
/usr/include/stdlib.h:735:12: note: candidate: int abs(int)
 extern int abs (int __x) __THROW __attribute__ ((__const__)) __wur;
            ^~~
In file included from /usr/include/c++/6/bits/stl_algo.h:59:0,
                 from /usr/include/c++/6/algorithm:62,
                 from ./_smach.h:38,
                 from ./smach.h:33,
                 from ../../src/gui/states/state_brush.h:33,
                 from states/state_brush.cpp:36:
/usr/include/c++/6/cstdlib:185:3: note: candidate: __int128 std::abs(__int128)
   abs(__GLIBCXX_TYPE_INT_N_0 __x) { return __x >= 0 ? __x : -__x; }
   ^~~
/usr/include/c++/6/cstdlib:180:3: note: candidate: long long int std::abs(long long int)
   abs(long long __x) { return __builtin_llabs (__x); }
   ^~~
/usr/include/c++/6/cstdlib:172:3: note: candidate: long int std::abs(long int)
   abs(long __i) { return __builtin_labs(__i); }
   ^~~
```

P.S. I build using this docker image - https://github.com/multiarch/crossbuild

--- a/synfig-studio/src/brushlib/brush.hpp
+++ b/synfig-studio/src/brushlib/brush.hpp
@@ -615,7 +615,7 @@ public:
 
       tilt_ascension = 180.0*atan2(-xtilt, ytilt)/M_PI;
       float e;
-      if (abs(xtilt) > abs(ytilt)) {
+	  if (std::fabs(xtilt) > std::fabs(ytilt)) {
         e = sqrt(1+ytilt*ytilt);
       } else {
         e = sqrt(1+xtilt*xtilt);

